src/client/components/ModalWithFeedback/ModalWithFeedback.css

```diff
@@ -0,0 +1,59 @@
+.feedback-modal .modal-wrapper {
+  width: 45%;
+  margin: 0;
+  height: 400px;
+  padding-bottom: 7%;
+}
+
+.dot {
+  height: 80px;
+  width: 80px;
+  background-color: #293a7d;
+  background-image: url('data:image/svg+xml;utf8,<svg width="46" height="34" viewBox="0 0 46 34" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M15.6109 25.2947L5.30172 15.4492L0.875 19.6768L15.6109 33.75L46 4.72763L41.5733 0.5L15.6109 25.2947Z" fill="%23ffff"/></svg>');
+  border-radius: 50%;
+  display: inline-block;
+  margin-left: 3%;
+  margin-right: 5%;
+  background-size: 70%;
+  background-repeat: no-repeat;
+  background-position: center top 50%;
+}
+
+.event-small-container {
+  background-color: #f6f6f6;
+  width: 90%;
+  display: flex;
+  box-sizing: border-box;
+  border-radius: 8px;
+  flex: wrap;
+  margin-left: 4%;
+  align-items: center;
+}
+
+.event-small-container .eventcardtext {
+  padding: 10px;
+}
+
+.feedback-textfield {
+  display: flex;
+  width: 100%;
+  margin: 4% 0% 5% 5%;
+  margin-left: 4%;
+}
+
+.feedback-textfield .input-component {
+  width: 90%;
+  font-size: 14px;
+  padding: 5%;
+  height: 5rem;
+}
+
+.feedback-submit-btn {
+  text-align: center;
```

Comment:
Hi Prabin, you need to change the button style and also the components should be placed on center. 
<img width="772" alt="Screenshot 2020-12-14 at 12 05 21" src="https://user-images.githubusercontent.com/61879499/102074177-a7be7400-3e04-11eb-8f25-030de3b1889b.png">

Reply:
there have been update with button components, modal components with styles.css , so i am waiting for things to get merge.